<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Отсекатель</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ввод мышкой</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Ввод отрезков</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Завершить ввод</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Отсеченеый отрезок</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Ввести отрезок</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Отрезок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ввести точку отсекателя</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Ввод отсекателя</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Клавиатурный ввод</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Отсечь</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Очистить экран</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="10" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Отсекатель</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ввод мышкой</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Ввод отрезков</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_10">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Завершить ввод</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Отсеченеый отрезок</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>Ввести отрезок</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>Отрезок</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_14">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ввести точку отсекателя</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Ввод отсекателя</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Клавиатурный ввод</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Отсечь</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>Очистить экран</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
